mk/docs.mk

```diff
@@ -287,12 +287,12 @@ doc/book/index.html: $(RUSTBOOK_EXE) $(wildcard $(S)/src/doc/trpl/*.md) | doc/
 	$(Q)rm -rf doc/book
 	$(Q)$(RUSTBOOK) build $(S)src/doc/trpl doc/book
 
-tarpl: doc/adv-book/index.html
+nomicon: doc/nomicon/index.html
 
-doc/adv-book/index.html: $(RUSTBOOK_EXE) $(wildcard $(S)/src/doc/tarpl/*.md) | doc/
+doc/nomicon/index.html: $(RUSTBOOK_EXE) $(wildcard $(S)/src/doc/nomicon/*.md) | doc/
 	@$(call E, rustbook: $@)
-	$(Q)rm -rf doc/adv-book
-	$(Q)$(RUSTBOOK) build $(S)src/doc/tarpl doc/adv-book
+	$(Q)rm -rf doc/nomicon
+	$(Q)$(RUSTBOOK) build $(S)src/doc/nomicon doc/nomicon
```

Comment:
Yes -- I personally felt `rustonomicon` was a bit too long, and also expect people to mess up the vowel after `rust`. I also don't expect much cold-typing of the URL. just autocompleteing/linking/bookmark/search.